### ISSUE TITLE
Enhancements needed to subtitle parser test suite.

### ISSUE THREAD

**Issue by ILOVEPIE** [SABRE-JS/SABRE.js#93]
We need more tests for the parsing code, both unit and comprehensive tests. Comprehensive tests can be made using [Aegisub](https://aegisub.org/). I am willing to peer program with/work with/mentor anyone who tackles this issue for hacktoberfest as it's a little more complex than some of the other ones we have available. Background as a typesetter for subtitles is useful but not needed.

**Comment by Aps2612** [comment]
Can you please assign me this ?

**Comment by ILOVEPIE** [comment]
> Can you please assign me this ?

Yes, but you might want to work with me on this one, so I can help you determine what tests you'll need to make. I'm pie_flavord on discord.
